README.md

```diff
@@ -1,4 +1,4 @@
-# envvar
+# envvar [![CircleCI](https://img.shields.io/circleci/project/github/plaid/envvar.svg)](https://circleci.com/gh/plaid/envvar)
```

Comment:
The paths should include the default branch name (`master`). Please copy the Sanctuary example. :)